app/src/main/java/net/skyscanner/backpack/demo/compose/BackpackPreview.kt

```diff
@@ -39,8 +38,8 @@ fun BackpackPreview(
   BackpackDemoTheme {
     CompositionLocalProvider(*providers) {
       Surface(
-        modifier = modifier
-          .background(background.takeOrElse { BpkTheme.colors.background }),
+        color = background.takeOrElse { BpkTheme.colors.background },
```

Comment:
we were setting the color via the modifier which meant it wasn't actually picked up correctly - set color directly